scripts/extensions/sams/src/components/assets/assetEditor.tsx

```diff
@@ -1,6 +1,9 @@
 // External modules
 import * as React from 'react';
 import {connect} from 'react-redux';
+import {OrderedMap} from 'immutable';
```

Comment:
Why use `OrderedMap` instead of an `Array`

Reply:
It is being converted into array below, where this function is called, line 188.

Reply:
```
description: ""
filename: "City7.jpg"
length: 1234818
mimetype: "image/jpeg"
name: "City7.jpg"
set_id: "602d04c5ee05b7bb601ebabf"
state: "draft"
tags: {name: "item6", code: "item6"}
```
This is the response without [ ].

Reply:
I'm still confused here, `tags` should always be an Array.